features/avatar-uploader/avatar-uploader.tsx

```diff
@@ -153,20 +164,22 @@ export const AvatarUploader: FC<AvatarUploaderProps> = ({
           type={'file'}
         />
         <div className={s.buttonsGroup}>
-          <Button
-            onClick={() => inputRef.current?.click()}
-            variant={previewOrAvatar ? 'tertiary' : 'primary'}
-          >
-            {t.buttons.select}
-          </Button>
+          {!previewOrAvatar || (
```

Comment:
Bug introduced! The button that allows uploading user's photo is not displayed if there is no already uploaded photo.